library/Fum2GitHub/Util.hs

```diff
@@ -0,0 +1,13 @@
+module Fum2GitHub.Util (
```

Comment:
I'd name this module `Fum2Github.URL` to prevent it from become total mess, as everything is `Util`.